src/test/java/at/ac/tuwien/kr/alpha/solver/OmigaBenchmarksTest.java

```diff
@@ -74,19 +77,19 @@ public void testLocstrat_400() throws IOException {
 		test("locstrat", "locstrat-400.txt");
 	}
 
-	@Test(timeout = 10000)
-	@Ignore("disabled to save resources during CI")
+	@Test(timeout = 15000)
 	public void testReach_1() throws IOException {
 		test("reach", "reach-1.txt");
 	}
 
-	@Test(timeout = 10000)
+	@Test(timeout = 15000)
```

Comment:
Why increase the timeout and at the same time ignore the test completely?

Reply:
OK, I take it back